### DISCUSSION TITLE
pico-font-size css variable does not work

### DISCUSSION THREAD

**Discussion by mostafatouny** [picocss/pico#647] [Help] [answered]
```
<style>
  nav a {
    font-size: 1.50rem;
  }
  td {
    text-align: center;
    font-size: 1.5rem;
    }
</style>
```
The css variable `--pico-font-size` documented [here](https://picocss.com/docs/css-variables#all-css-variables) did not. This hack workaround did it.

1. Is that a bug?
2. Is my workaround safe? Recommended alternatives?

**Comment by dev-willis** [comment, marked as answer]
It's not a bug; you probably just didn't use the necessary specificity. It's fine to do it this way too tho. And there's nothing you can do in CSS that is unsafe.

**Reply by mostafatouny** [reply]
Thank you. Do you recommend me to learn anything, to be able to use the right specificity?
